auslib/test/admin/views/test_releases.py

```diff
@@ -865,3 +865,11 @@ def testReadOnlyNonAdminCanSetFlagButNotUnset(self):
         data_unset = dict(name='b', read_only='', product='Firefox', data_version=2)
         ret = self._put('/releases/b/read_only', username='bob', data=data_unset)
         self.assertStatusCode(ret, 403)
+
+
+class TestSingleColumn_JSON(ViewTest, JSONTestMixin):
+
+    def testGetReleasesSingleColumn(self):
+        ret = self._get("/releases/column/product")
+        got = json.loads(ret.data)
+        self.assertEquals(got["count"], 4)
```

Comment:
Please check the entire return value here, not just the count. I'd also like to see an additional test that makes sure that filtering on a column that doesn't exist causes a 404.